src/ui/public/doc_table/components/table_row.js

```diff
@@ -158,6 +163,22 @@ module.directive('kbnTableRow', function ($compile) {
 
         return text;
       }
+
+      function truncateSourceSummary(sourceElement) {
```

Comment:
Seems to me it might make more sense to have this logic [here](https://github.com/elastic/kibana/blob/6e819c11b87c7c6a96759ba018026e7bb159c1a9/src/ui/public/stringify/types/source.js#L28). Then you don't have to check if the `"discover-table-sourcefield"` class exists, nor deal with the DOM. What do you think?